discovery-infra/tests/test_dhcp.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from tests.conftest import env_variables, is_qe_env
+from tests.base_test import BaseTest
+
+
+@pytest.mark.skipif(not env_variables['vip_dhcp_allocation'], reason="DHCP allocation must be enabled")
```

Comment:
This means DHCP and none DHCP cannot run at the same run,
Need to find a way to set it up in test level and not using the vip_dhcp_allocation

Reply:
How about adding the ability, to override the cluster creation params in the fitxture:
  https://github.com/openshift/assisted-test-infra/blob/580aea5aab7c6b822d5719b2862e5f175e53062b/discovery-infra/tests/base_test.py#L37

Reply:
This test isn't relevant on non-DHCP mode

Reply:
yes, but the mode defined in the cluster level

Reply:
currently the mode is depended on the env vip_dhcp_allocation, so each test run has true or false, meaning we wont be able to run both DHCP and none DHCP tests, and we do want that

Reply:
It would require a bigger change to the infrastrucutre. For now, I just made sure the test wouldn't run with inappropriate configuration.

Reply:
The test wont run unless youll explicitly run it, it wont run as part of the regression, so what is the point?
It is not a big change, and it is required

Reply:
We can remove VIP_DHCP_ALLOCATION env variable all together and default vip_dhcp_allocation=false durring cluster creation and allow to override this